test-clients/scripts/regressionFlowUpdateFiles.sh

```diff
@@ -61,4 +61,26 @@ updateServiceMainJava()
 
 }
 
-updateServiceMainJava
+PLATFORM_REGRESSION_REPO_PATH=""
```

Comment:
I this used somewhere ?

Reply:
Yes it is used in line 71

Reply:
`cp "$PLATFORM_REGRESSION_PATH/tmp/config.txt" $TARGET_DIR` ??

Reply:
Oh you are correct. It is not used anywhere. Will remove it. Thank you!!